components/dialog/dialog-fullscreen.js

```diff
@@ -84,6 +84,12 @@ class DialogFullscreen extends LocalizeCoreElement(AsyncContainerMixin(DialogMix
 					width: auto;
 				}
 
+				dialog.d2l-dialog-outer.d2l-dialog-fullscreen-within,
+				div.d2l-dialog-outer.d2l-dialog-fullscreen-within {
+					/* no margins when there is a fullscreen element within */
+					margin: 0;
```

Comment:
https://github.com/BrightspaceUI/core/pull/2195/files#diff-e9552afeeb4d9451d8ff1940942a2414163e00d1cb2572664f60cd6d5ddc7187R78 Defines a 1.5rem margin around the dialog for a border. If there is a fullscreen element within (like HTML editor), we don't want the margins. Note that this fix is only for the wide screen layout, as the mobile layout was using regular `d2l-dialog` styles that didn't have this issue.